integration/k8s-client.go

```diff
@@ -109,16 +109,26 @@ func GetStatefulsetList(ctx context.Context, ns string, client kubernetes.Interf
 	return sfsList.Items, err
 }
 
+// GetVolume returns a volume given a namespace and name
+func GetVolume(ctx context.Context, ns, name string, client kubernetes.Interface) (*corev1.PersistentVolumeClaim, error) {
```

Comment:
Is this being used?

Reply:
@maroshii https://github.com/okteto/okteto/pull/4204/files#diff-57626eb367add33c287ac942e1a7a64fa1181b54d59f0aaf5647957c8b9b92b7R238 and another instance below